oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -318,6 +318,9 @@ Post a message to `Open edX Discourse Deprecation Announcements`_, using the fol
         that <*techonology name*> is valuable and wish to make a case for retaining
         it and taking over its maintainership, please speak up on the issue linked above.
 
+        After acceptance, you must watch the Github issue for any future notifications
```

Comment:
"you must" is a bit direct. I wonder if "After acceptance, all future notifications around removal will only be posted to the GitHub issue, so be sure to watch that issue for updates" would sound nicer.